opentelemetry-jaeger-propagator/src/propagator.rs

```diff
@@ -82,10 +81,11 @@ impl Propagator {
 
         let parts = header_value.split_terminator(':').collect::<Vec<&str>>();
         if parts.len() != 4 {
-            global::handle_error(Error::Propagation(PropagationError::extract(
-                "invalid jaeger header format",
-                "JaegerPropagator",
-            )));
+            otel_warn!(
```

Comment:
I haven't followed the entire code path, so unclear what is the impact of this Warning. For eg: if an end users see this log entry, what should they be doing? 
For all user facing logs, we should be very clear about the impact of this - will a span be dropped? will a span be using different parent ? will a span be created as root span? will we just ignore the header for tracestate and baggage too? or just ignore the traceparent header? something else?

I am not saying this should be fixed in this PR, as these are existing issues, nothing newly introduced in this PR. Since Metrics, Logs are where we are actively working towards Stable, we can probably merge this now, and come back when we start working towards Tracing Stable release?